### PR TITLE
DXCDT-283: Remove `config` command

### PR DESCRIPTION
**Pull request by willvedd**
### 🔧 Changes

Removes the `config` command. This command was undocumented and only existed to instantiate a tenant for integration tests but this functionality already exists with the `auth0 tenants add` command. Further, it was undocumented and not meant to be utilized by users. 

This PR accomplishes a few things:
- Have our integration tests follow the actual, idiomatic authentication mechanisms 
- Set us up for a cleaner transition when `auth0 login` and `auth0 tenants add` commands are consolidated
- Simplify the codebase

Originally, this command was added with #291 , at a time when `auth0 tenants add` did not get added until #304. 

### 📚 References

- Original `config` command PR: #291
- Tenants add command PR: #304

### 🔬 Testing

Confirms working if passes integration tests in CI.

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/cli/cli.go [hunk 1]
```diff
@@ -499,14 +499,6 @@ func defaultConfigPath() string {
 	return path.Join(os.Getenv("HOME"), ".config", "auth0", "config.json")
 }
 
-func (c *cli) setPath(p string) {
```
Comment (willvedd) [inline internal/cli/cli.go:502]: Only used by config command.

File: internal/cli/tenants.go [hunk 2]
```diff
@@ -203,6 +203,9 @@ func addTenantCmd(cli *cli) *cobra.Command {
 		},
 	}
 
+	tenantClientID.RegisterString(cmd, &inputs.ClientID, "")
+	tenantClientSecret.RegisterString(cmd, &inputs.ClientSecret, "")
```
Comment (willvedd) [inline internal/cli/tenants.go:207]: The `auth0 tenants add` command never respected the usage of of client ID and client secret flags. This fixes that.

**Comment by willvedd** [comment]
Ack'ing that [integration tests are failing](https://github.com/auth0/auth0-cli/actions/runs/3578136331/jobs/6017971855) because `api` command does not support client credentials as a form of authentication. That will need to be solved before this can get merged in.

**Comment by willvedd** [comment]
Integration tests finally passing now that #537 has been incorporated.

**Review by Widcket — APPROVED** [review]

**Review by sergiught — APPROVED** [review]

**Pull request merged**